backend/db/migrate/20220516085522_add_impact_demands_to_locations.rb

```diff
@@ -0,0 +1,13 @@
+class AddImpactDemandsToLocations < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension "postgis"
```

Comment:
I'm not sure if that will work with migration on staging (if application user will have enough rights)

Reply:
In the end `enable_extension` just worked in migration, which is a surprise, but I'm not going to complain. It is true that with newer version of postgres you no longer need superuser to create extensions, but still surprising to see anything *just work* here.